.github/actions/ci/action.yml

```diff
@@ -8,35 +8,72 @@ inputs:
 runs:
   using: composite
   steps:
-    - uses: leafo/gh-actions-lua@35bcb06abec04ec87df82e08caa84d545348536e
+    - name: Install Lua
+      uses: leafo/gh-actions-lua@35bcb06abec04ec87df82e08caa84d545348536e
       with:
         luaVersion: ${{ inputs.lua-version }}
-    - uses: leafo/gh-actions-luarocks@e65774a6386cb4f24e293dca7fc4ff89165b64c5
-    - name: Install C Server-side SDK Dependencies
-      shell: bash
-      run: |
-        sudo apt-get update -y
-        sudo apt-get install -y libcurl4-openssl-dev
-        sudo apt-get install -y libhiredis-dev
-
-    - name: Setup C Server-side SDK
-      shell: bash
-      run: ./scripts/build-c-server.sh
+    - name: Install LuaRocks
+      uses: leafo/gh-actions-luarocks@e65774a6386cb4f24e293dca7fc4ff89165b64c5
+    - name: Install Boost
+      id: install-boost
+      uses: MarkusJx/install-boost@v2.4.4
+      with:
+        boost_version: 1.81.0
+        platform_version: "22.04"
+    - name: Install CPP SDK
+      uses: ./.github/actions/install-cpp-sdk-redis
+      with:
+          version: launchdarkly-cpp-server-redis-source-v2.1.0
```

Comment:
This pulls in the C++ server's prebuilt Linux artifacts. It saves us a bunch of time compiling the SDK manually in the runner.